tests/baseline/cases/staged-simple-ami.yml

```diff
@@ -44,6 +44,7 @@ items:
     sriov_net_support: simple
     state: PENDING
     type: access
+    uefi_support: null
```

Comment:
Can we please have at least one case where this field isn't using the default/null value?

Without that we're not testing that the value is being read and propagated through the library, e.g. the tests would still pass if you'd forgotten the `"uefi_support": data.get("uefi_support"),` bit or if someone breaks it later.